features/tweet.feature

```diff
@@ -0,0 +1,10 @@
+Feature: Tweet
+
+  Scenario: Tweet Creation
```

Comment:
If part of the purpose of integration testing is to mimic a user's behavior, the user should know what they are typing. Maybe something more like:
```
Scenario: Tweet Creation
  Given I am logged into the site
   When I visit the "/users/following" page
   And I click "Create Tweet"
   And I fill in "Text" with "Look at me I'm tweeting!"
   And I click "Create Tweet"
   And I visit my own profile page
   Then I should see "Look at me I'm tweeting!"
```
The syntax might be a little off from your step definitions. This is just something to think about, not a necessary change.